### PR TITLE
UI/Custom Icons for Nodes

### PR DESCRIPTION
**Pull request by abhishekshankr**
Updated icon set for a majority of nodes with custom icons to make nodes consistent.

image

### CODE REVIEW

File: packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ class OpenAIAssistant_Agents implements INode {
         this.version = 2.0
         this.type = 'OpenAIAssistant'
         this.category = 'Agents'
-        this.icon = 'openai.png'
+        this.icon = 'assistant.svg'
```
Comment (HenryHengZJ) [inline packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts:30]: For this icon, is it possible to add OpenAI logo that cuts into the bot icon? Reason being this is specific to OpenAI only

File: packages/components/nodes/agents/OpenAIFunctionAgent/OpenAIFunctionAgent.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ class OpenAIFunctionAgent_Agents implements INode {
         this.version = 3.0
         this.type = 'AgentExecutor'
         this.category = 'Agents'
-        this.icon = 'openai.png'
+        this.icon = 'function.svg'
```
Comment (HenryHengZJ) [inline packages/components/nodes/agents/OpenAIFunctionAgent/OpenAIFunctionAgent.ts:26]: same for this as well

File: packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts (hunk 1 above)
Comment (abhishekshankr) [reply packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts:30]: Updated the icon with OpenAI logo

File: packages/components/nodes/agents/OpenAIFunctionAgent/OpenAIFunctionAgent.ts (hunk 2 above)
Comment (abhishekshankr) [reply packages/components/nodes/agents/OpenAIFunctionAgent/OpenAIFunctionAgent.ts:26]: Updated the icon with OpenAI logo

**Comment by HenryHengZJ** [comment]
thank you @abhishekshankr !!

**Pull request merged**